itests/direct_data_onboard_test.go

```diff
@@ -140,6 +145,23 @@ func TestOnboardRawPieceVerified(t *testing.T) {
 		kit.Account(verifiedClientKey, abi.NewTokenAmount(bal.Int64())),
 	)
 
+	evtChan, err := miner.FullNode.SubscribeActorEvents(ctx, &types.SubActorEventFilter{
+		Filter: types.ActorEventFilter{
+			MinEpoch: -1,
+			MaxEpoch: -1,
+		},
+		Prefill: true,
+	})
+	require.NoError(t, err)
+
+	events := make([]types.ActorEvent, 0)
+	go func() {
+		for e := range evtChan {
+			fmt.Printf("%s Got ActorEvent: %+v", time.Now().Format(time.StampMilli), e)
```

Comment:
currently doesn't show anything because they get filtered out due to address translation in the event filter

Reply:
Fixed